### ISSUE TITLE
It is not clear which content pane is selected

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-ev3#669]
**Repro steps:**
• Open https://makecode.mindstorms.com/beta
• Click "More..." -> "High Contrast On" mode
• Observe "Blocks" & "{ }JavaScript" buttons on menu bar
**Expected result:**
It should be clear which content pane is selected
**Actual result:**
It is not clear which content pane is selected
![yyy](https://user-images.githubusercontent.com/23466737/40776333-03d3172c-64ba-11e8-8bf1-215bd8e4b53d.PNG)
